mftf/2.3/debugging.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Debugging MFTF Tests
+mftf-release: 2.3.13
+---
+
+Debugging within the Magento Functional Testing Framework is helpful in identifying test bugs by allowing you to pause execution so that you may:
+
+- Examine the page.
+- Check returned data and other variables being used during run-time.
+
+This is straightforward to do once you create a basic Debug Configuration.
+
+## Prerequisites
+
+- [Xdebug]
+- PHPUnit configured for use in [PHPStorm]
+
+## Creating Debug Configuration with PHPStorm
+
+1. Download the Codeception Framework plugin for PHPStorm (`PhpStorm->Preferences->Plugins`).
+1. Click `Edit Configurations` on the configuration dropdown.
+1. Click `+` and select `Codeception` from the available types.
+1. Find the `Custom Working Directory` option and set the path to your `dev/tests/acceptance/` directory.
```

Comment:
I feel like something is missing here because my configuration is still not ready to run. The window is asking for information to be filled out in the Test Runner section (at the top of the configuration window).

Reply:
@dobooth he is correct, need to add a step for:
``` 1. Change `Test Scope` to `Type` and select `functional` from the `Type:` dropdown
```